.gitignore

```diff
@@ -33,4 +33,13 @@ lerna-debug.log*
 !.vscode/settings.json
 !.vscode/tasks.json
 !.vscode/launch.json
-!.vscode/extensions.json
\ No newline at end of file
+!.vscode/extensions.json
+
+# yarn
+**/.pnp.*
+**/.yarn/*
+!**/.yarn/patches
+!**/.yarn/plugins
+!**/.yarn/releases
+!**/.yarn/sdks
+!**/.yarn/versions
```

Comment:
These exceptions are not needed anymore, right?

Reply:
You can still add patches and such this way with this version of yarn.

Reply:
So they're not needed right now but they might be needed in the future

@genaris let me know if you want me to remove them

Reply:
No problem!